openedx/features/marketplace/models.py

```diff
@@ -1,6 +1,33 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from django.contrib.auth.models import User
 from django.db import models
 
-# Create your models here.
+from openedx.custom.db.fields import MultiSelectWithOtherField
+from openedx.features.idea.models import VisualAttachment, Location, OrganizationBase
+from openedx.features.marketplace.constants import (ORGANIZATION_SECTOR_CHOICES, ORGANIZATIONAL_PROBLEM_CHOICES,
+                                                    USER_SERVICES)
+
+
+class Challenge(OrganizationBase, Location, VisualAttachment):
```

Comment:
is it a `Challenge` or `MarketplaceRequest` I am not suer in figma they have used Merketplace everywhere. @emraan004 @saad-waqar WDYT?

Reply:
even the name of this app is marketplace.

Reply:
@saad-waqar , @emraan004 what should be the name, as I feel Challenge a more relevant. but I would also like to ask if it is fine ??

Reply:
I had the same point in mind. Actually Both are right I guess. When the client gave the initial demo of this feature they were calling these challenges. Now in the figma its called "request" everywhere. Honestly I'm inclined towards MarketplaceRequest

Reply:
update to `MarketplaceRequest `